WebDriverAgentLib/Commands/FBSessionCommands.m

```diff
@@ -40,6 +40,8 @@
 static NSString* const DEFAULT_ACTIVE_APPLICATION = @"defaultActiveApplication";
 static NSString* const ACTIVE_APP_DETECTION_POINT = @"activeAppDetectionPoint";
 static NSString* const INCLUDE_NON_MODAL_ELEMENTS = @"includeNonModalElements";
+static NSString* const ACCEPT_ALERT_BUTTON_LOCATOR = @"acceptAlertButtonLocator";
```

Comment:
'Selector' instead of 'Locator'?  (other places, too)
Locator is '-ios class chain' part in find element and selector is the given value to search with.
https://appiumpro.com/editions/20

Reply:
changed